shopping-cart/.gitignore

```diff
@@ -0,0 +1,5 @@
+/bin/
+/.idea/
+/.vscode/
+/target/
+*.iml
```

Comment:
There is .gitignore in the root folder.

Reply:
Ah - sorry - will merge.

Reply:
Changes pushed - ready for another review.